options/internal/aarch64/gcc-extra/cxxabi.cpp

```diff
@@ -0,0 +1,11 @@
+
+#include <bits/ensure.h>
+
+// The cxxabi needs operator delete for *deleting* destructors, i.e., destructors that
+// are called by delete expressions. We never use such expressions in mlibc.
+// Note that G++ complains if we make the operator hidden,
+// thus we use it's mangled name as a workaround.
+extern "C" [[gnu::visibility("hidden")]] void _ZdlPvm() { // operator delete (void *, size_t)
```

Comment:
Is this different from the x86_64 case?

Reply:
All the files in that subdir were the same so I just use the top-level copy instead now.

Reply:
Update: I didn't notice `mlibc_crtend.S` differs because it contains actual asm.